azure-pipelines.yml

```diff
@@ -48,7 +50,7 @@ jobs:
       package_readme: null
     before:
       - script: 'sudo apt-get update -y && sudo apt-get install fish csh'
-        condition: and(succeeded(), eq(variables['image_name'], 'linux'), in(variables['TOXENV'], 'py37', 'py36', 'py35', 'py34', 'py27'))
+        condition: and(succeeded(), eq(variables['image_name'], 'linux'), in(variables['TOXENV'], 'py38', 'py37', 'py36', 'py35', 'py34', 'py27'))
         displayName: install fish and csh via apt-get
       - script: 'brew update -vvv && brew install fish tcsh'
         condition: and(succeeded(), eq(variables['image_name'], 'macOs'), in(variables['TOXENV'], 'py37', 'py36', 'py35', 'py34', 'py27'))
```

Comment:
@gaborbernat Need to add `'py38'` to this conditional to fix that `py38 macOS` fail.

Reply:
thanks!